src/libopensc/card-myeid.c

```diff
@@ -131,9 +135,13 @@ static int myeid_init(struct sc_card *card)
 	u8 defatr[SC_MAX_ATR_SIZE];
 	size_t len = sizeof(defatr);
 	const char *atrp = myeid_atrs[MYEID_INFINEON_CHIP_ATR];
-
+	
 	LOG_FUNC_CALLED(card->ctx);
+
+	card->name = myeid_card_name;
```

Comment:
OK, then nevermind the card name. However, you still need to rebase onto the current master. git shell on windows is no different than on linux. I'm afraid you will need to get familiar with it. here are some links that may be interesting http://chris.beams.io/posts/git-commit/ https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History